webapp/src/main/webapp/news-details/components/ExoNewsDetails.vue

```diff
@@ -187,13 +192,7 @@ export default {
       };
       socialProfile.initUserProfilePopup('newsDetails', labels);
     });
-    const linkContentElements = document.querySelectorAll('#newsDetails a');
-    linkContentElements.forEach(function(linkContentElement) {
-      if (linkContentElement && !linkContentElement.href.includes(`${document.location.host}${eXo.env.portal.context}`)) {
-        linkContentElement.setAttribute('target', '_blank');
-      }
-    });
-    
+    this.news.body = this.targetBlank(this.news.body);
```

Comment:
We need also normally the same treatment for summary.
Why we need to change this treatment ?